### ISSUE TITLE
Совместимость sql

### ISSUE THREAD

**Issue by zhalil** [Restream/reindexer#84]
Совместим ли reindexer с mysql? получится ли как в тарантуле просто воспользоваться mysql драйвером для работы с реиндексером?

**Comment by slowcheetahzzz** [comment]
Нет, автоматически привязать reindexer к определенным полям таблиц mysql (как и любой другой СУБД) как в Tarantool невозможно.
